packages/recomponents/src/components/r-date-range/r-date-range.vue

```diff
@@ -328,8 +328,14 @@
                 // so we have to convert
                 // the problem details: if your browser timezone is +3 then you will have .toISOString as 21:00
                 // and user profile timezone will never change that value
-                const start = this.timezoneHandler().getRaw(moment(date.start).startOf('day'));
-                const end = this.timezoneHandler().getRaw(moment(date.end).endOf('day'));
+                const start = this.minDate && this.checkDatesAreSame(this.minDate, date.start)
```

Comment:
Should we also clamp the date.start to the minDate (right now it seems if the date.start is before the minDate this would still use the date.start and not the minDate)?

same for maxDate but for after instead

Reply:
The picker will not allow selecting the date before the `minDate` or after the `maxDate`